lib/fixbase.ml

```diff
@@ -439,3 +439,92 @@ let fix_key ?report progress base =
       ignore @@ loop [] rev_list
     end
   end ipers
+
+(* Scan a base to identify potential conflicts arising when:
+   - replacing ’ by ' in the lower function
+   - properly treating supplementary Latin accented characters (for vietnameese)
+   resolution typically consists in changing the occ number (+1)
+*)
+
+let special_utf_8 s =
+  let s = match Char.code s.[0] with
+    | 0xE2 when String.length s = 3 &&
+            Char.code s.[1] = 0x80 &&
+           (Char.code s.[2] = 0x98 || (* ’ apostrophes typo *)
+            Char.code s.[2] = 0x99)   (* ‘ autre apostrophe *)
+          -> " "
+    | _ -> s
```

Comment:
since we only match on `0xE2` and have `when ...` condition in the match, maybe use a `if ... then ...` instead

Reply:
Thanks for the comment. Updated accordingly